components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1993,17 +1993,16 @@ def define(self):
         args = {
             "constructorBehavior": constructorBehavior,
             "id": name,
-            "representation": str_to_const_array("function %s() {\\n    [native code]\\n}" % name),
+            "representation": "b\"function %s() {\\n    [native code]\\n}\"" % name,
```

Comment:
Use single quotes for the python string to avoid escaping the Rust quotes; r+ with that